File: 2021-07/2021-07-23 (day5)/CXPhoenix/solution1.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        l = 0
        r = len(nums)-1
        while l <= r:
            mid = (r+l)//2
            if target > nums[mid]:
                l = mid +1
            else:
                r = mid - 1
        return l
        

File: 2021-08/2021-08-05 (day18)/A_GO/solution1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> output;
        for(int i=0;i < nums.size();i++){
            for(int j=i+1; j<nums.size(); j++){
                if((nums[i] + nums[j]) == target){
                    output.push_back(i);
                    output.push_back(j);
                    break;
                }
            }
        }
        return output;
    }
};